modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/cxf/clientConfig/CXFClientConfigElytronNotOnClasspathTestCase.java

```diff
@@ -0,0 +1,140 @@
+package org.jboss.test.ws.jaxws.cxf.clientConfig;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.wsf.stack.cxf.client.ProviderImpl;
+import org.jboss.wsf.test.JBossWSTest;
+import org.jboss.wsf.test.JBossWSTestHelper;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.StringTokenizer;
+
+@RunWith(Arquillian.class)
+public class CXFClientConfigElytronNotOnClasspathTestCase extends JBossWSTest {
+   private final String FS = System.getProperty("file.separator"); // '/' on unix, '\' on windows
+
+   @ArquillianResource
+   private URL baseURL;
+
+   public static final String CLIENT_JAR = JBossWSTestHelper.writeToFile(new JBossWSTestHelper.JarDeployment("jaxws-cxf-jbws3713-client.jar") {
```

Comment:
could we use other name for JAR, eg. 'jaxws-cxf-jbws4179-client.jar'? It is already used by other tests

Reply:
Fixed.